doc/tutorials/understanding-error-messages.md

```diff
@@ -17,8 +17,19 @@ sudo kdb set system/tests/hosts/ipv4/my 88.198.134.1777
 sudo kdb umount system/tests/hosts
 ```
 
+In this example we mount the hosts file under `/etc/hosts` into Elektra. The `hosts` plugin will also validate
```

Comment:
It is not the hosts plugin itself, please look how it works (glob plugin marks with metadata, and ipaddr actually validates). This is very different to the specification model. For specifications, see doc/tutorials/validation.md

This tutorial could also cover how to specify check/validation/message.

Reply:
I rewrote the sentence